<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="TYC0522054" failures="2" tests="4" name="Test" time="52.27" errors="0" timestamp="2024-08-24T11:44:38 IST">
  <testcase classname="testCases.TC_003_LoginDDT" name="verify_loginDDT" time="5.711"/>
  <testcase classname="testCases.TC_003_LoginDDT" name="verify_loginDDT" time="12.13"/>
  <testcase classname="testCases.TC_003_LoginDDT" name="verify_loginDDT" time="11.613">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_003_LoginDDT.verify_loginDDT(TC_003_LoginDDT.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_003_LoginDDT" name="verify_loginDDT" time="11.591">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_003_LoginDDT.verify_loginDDT(TC_003_LoginDDT.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
</testsuite> <!-- Test -->
